apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -224,8 +224,24 @@ function CourseNewRequestCard({ csrfToken }: { csrfToken: string }): HtmlValue {
             <label class="form-label" for="cr-ghuser">GitHub Username (optional)</label>
             <input type="text" class="form-control" name="cr-ghuser" id="cr-ghuser" />
             <small class="form-text text-muted">
-              Providing your GitHub username will allow you to edit course content offline. You do
-              not need to provide this if you would like to use the online web editor.
+              Providing your GitHub username will allow you to edit course content offline or run a
```

Comment:
I think turning this into a bulleted list would be helpful.

```
Providing your GitHub username will give you access to the GitHub repo storing your course content.

This allows you to:

- edit course content offline
- run a local installation of PrairieLearn
- plan complex questions, including questions that use external grading (e.g. programming questions) or workspaces
- manage access to your GitHub repo, e.g. to add other instructors or TAs

You do not need to provide this if you would like to use the online web editor.
```

Reply:
If we want to talk about this in more depth, maybe adding/updating some info in the docs and then linking to it from here?

Reply:
Added the section in 0db64982aea1ec7659ef5ad097aaa91e9b26e6c2.